src/pages/Commits.tsx

```diff
@@ -40,6 +43,15 @@ export const Commits = () => {
   const dispatchToast = useToastContext();
   const { replace } = useHistory();
   const { search } = useLocation();
+  const hasSeenChartToggle = Boolean(Cookies.get(HAS_SEEN_COMMIT_CHART_TOGGLE));
+  const [isOpenChartToggle, setIsOpenChartToggle] = useState(
+    !hasSeenChartToggle
+  );
+  useEffect(() => {
+    if (!hasSeenChartToggle && !isOpenChartToggle) {
+      Cookies.set(HAS_SEEN_COMMIT_CHART_TOGGLE, "true");
+    }
+  }, [hasSeenChartToggle, isOpenChartToggle]);
```

Comment:
```suggestion
  const shouldChartOpen = Boolean(Cookies.get(COMMIT_CHART_TOGGLE));
const onToggleChart = ({isVisible) => {
Cookies.set(COMMIT_CHART_TOGGLE, isVisible)
```

Reply:
It makes more sense to store the last state the toggle was on.

Reply:
I remember in the design meeting we discussed that the chart should only be open to the user once and then closed after. I can run this by Lara again